.pre-commit-config.yaml

```diff
@@ -4,23 +4,20 @@ default_language_version:
 exclude: '^tools/(prettify/fprettify|build_utils/fypp)'
 fail_fast: false
 repos:
-- repo: https://github.com/pre-commit/mirrors-yapf
-  rev: v0.24.0
+- repo: https://github.com/ambv/black
+  rev: stable
   hooks:
-  - id: yapf
+  - id: black
```

Comment:
I'm very happy about switching from yapf to black!

Reply:
cool :)